src/Modules/CalcOffence.lua

```diff
@@ -4103,8 +4100,14 @@ function calcs.offence(env, actor, activeSkill)
 			output.CombinedDPS = output.CombinedDPS + activeSkill.mirage.output.TotalDot * (skillFlags.DotCanStack and mirageCount or 1)
 		end
 		if activeSkill.mirage.output.CullMultiplier > 1 then
-			output.MirageDPS = output.MirageDPS * (activeSkill.mirage.output.CullMultiplier - 1)
-			output.CombinedDPS = output.CombinedDPS * (activeSkill.mirage.output.CullMultiplier - 1)
+			mirageCulling = activeSkill.mirage.output.CullMultiplier - 1
 		end
 	end
+
+	local bestCull = 1
+	if output.CullMultiplier > 1 or mirageCulling and mirageCulling > 1 then
+		bestCull = m_max(output.CullMultiplier > 1 and output.CullMultiplier or 1, mirageCulling and mirageCulling > 1 and mirageCulling or 1)
```

Comment:
This could be simplified by just setting `mirageCulling` to 1 initially:

```suggestion
	if output.CullMultiplier > 1 or mirageCulling > 1 then
		bestCull = m_max(output.CullMultiplier, mirageCulling)
```